### ISSUE TITLE
Feature request: Add looping playback

### ISSUE THREAD

**Issue by wrldwzrd89** [benwiggy/APPlayMIDI#5]
Much like what QuickTime Player 7 offered for MIDIs, APPlayMIDI should offer loop play as an option (resetting the MIDI engine when the song completes if this is needed, even if doing so causes a pause in audio). I'd rather have the option of letting MIDIs play in the background without me having to queue them up / restart playing manually, if I elect to use MIDIs as background music.

**Comment by benwiggy** [comment]
Yes, it should be easy enough to implement a loop control for each document. I'm not going to offer looping through a "playlist" of successive MIDI files, though.

**Comment by wrldwzrd89** [comment]
Per-document loop control is good enough!

**Comment by Sanabria** [comment]
+1 please

**Comment by benwiggy** [comment]
v1.14 current build has a loop switch.
